mantidproject/mantid
# Add developer documentation GUI Guidelines for Icons

**Description of work.**
Add developer documentation GUI Guidelines for Icons

**To test:**
- Build target `dev-docs-html`
- Navigate to your `BuildDirectory/dev-docs/html/GUIDesignGuidelines.html`
- Check that the page looks ok.
- Click the `Mantid Table Icon` link in the icons section
- Check that page looks ok
- Doc review



*There is no associated issue.*


*No release notes as this is a developer doc*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
